README.md

```diff
@@ -29,17 +29,26 @@ let server = new QueryParamAwarePretender();
 
 server.get('/api/graphql?foo=bar', () => [ 200, {}, '{ "query": "bar" }'),
 server.get('/api/graphql?foo=baz', () => [ 200, {}, '{ "query": "baz" }'),
+server.get('/api/graphql?foo=*', () => [ 200, {}, '{ "query": "xyz" }'),
```

Comment:
Nit: All of these `server.get(...` lines seem to be missing a `]` (and, for that matter, they end in commas rather than semicolons) - totally unrelated to this PR and could be cleaned up separately

Reply:
Good catch!!!